common/types/verified_activation.go

```diff
@@ -0,0 +1,69 @@
+package types
+
+import "github.com/spacemeshos/go-spacemesh/log"
+
+type VerifiedActivationTx struct {
+	*ActivationTx
+
+	baseTickHeight uint64
+	tickCount      uint64
+}
+
+// GetWeight of the atx.
+func (vatx *VerifiedActivationTx) GetWeight() uint64 {
+	return uint64(vatx.NumUnits) * (vatx.tickCount)
```

Comment:
This might be a stupid question... I don't know the practical bounds of these variables yet :)

Can it, in practice, overflow here? It's a uint32 * uint64. Perhaps consider annotating in a comment why it is safe to multiply.

Similarly when you add two `uint64` in `TickHeight()`.